products/pnpm.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Pnpm
+category: app
+tags: package-manager javascript-runtime
+iconSlug: pnpm
+permalink: /pnpm
+versionCommand: pnpm --version
+releasePolicyLink: https://github.com/pnpm/pnpm/security/policy
+changelogTemplate: "https://github.com/pnpm/pnpm/releases/tag/__LATEST__"
+releaseDateColumn: true
+eolColumn: Support
+
+auto:
+  methods:
+  -   npm: pnpm
+
+releases:
+-   releaseCycle: "9"
+    releaseDate: 2024-04-16
+    eol: false
+    latest: '9.0.6'
+    latestReleaseDate: 2024-04-24
+
+-   releaseCycle: "8"
+    releaseDate: 2023-03-27
+    eol: 2025-04-30
+    latest: '8.15.7'
+    latestReleaseDate: 2024-04-13
+
+-   releaseCycle: "7"
+    releaseDate: 2022-04-30
+    eol: 2024-04-16
+    latest: '7.33.7'
+    latestReleaseDate: 2024-02-15
+
+-   releaseCycle: "6"
+    releaseDate: 2021-04-05
+    eol: 2023-03-20
+    latest: '6.35.1'
+    latestReleaseDate: 2022-11-11
+
+-   releaseCycle: "5"
+    releaseDate: 2020-05-27
+    eol: 2022-04-29
+    latest: '5.18.11'
+    latestReleaseDate: 2021-09-16
+
+-   releaseCycle: "4"
+    releaseDate: 2019-10-07
+    eol: 2021-04-24
+    latest: '4.14.4'
+    latestReleaseDate: 2020-05-22
+
+---
+
+> [Pnpm](https://pnpm.io/) is a fast, disk space efficient package manager for JavaScript.
+> It is a drop-in replacement for npm, but faster and more efficient.
```

Comment:
Could you add a little paragraph explaining the support policy ? From what I could see with the comment of @captn3m0 they support the last two releases ?

Reply:
upstream did not explicitly see that, but I guess we can assume it.

Reply:
My comment was a question, not an affirmation. It's also OK to just write that it's documented on https://github.com/pnpm/pnpm/security/policy.

Reply:
Ok, let's forget that: the releasePolicyLink is enough for now.